docs/cluster-configuration.md

```diff
@@ -109,3 +109,29 @@ tolerations.
           value: "kubevirt"
           effect: "NoSchedule"
   ```
+
+
+
+
+## Additional Configurations
```

Comment:
I find this section superfluous, adding very little info.. I'd promote featureGate section to level two (##)